layouts/posts/single.html

```diff
@@ -13,6 +13,7 @@ <h1>{{ .Title }}{{ if .Site.Params.addDot }}<span class="dot">.</span>{{ end }}<
     </div>
     <div class="content">
       {{ .Content }}
+      {{ partial "disqus.html" . }}
```

Comment:
wrap inside condition that checks if DisqusShortname is set, if it's not set we shouldnt render this partial at all.